ci-operator/step-registry/ipi/install/splay/ipi-install-splay-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-install-splay
+  from: cli
+  commands: ipi-install-splay-commands.sh
+  resources:
+    requests:
+      cpu: 1m
+      memory: 1Mi
+  documentation: |-
+    Injects a random splay period to prevent hitting quota issues on cloud providers like GCP.
```

Comment:
nit: drop "like GCP"?  Because you're currently just using this for Azure.  Maybe:

> Injects a random delay of up to 10 minutes to reduce install-time contention and subsequent cloud-provider throttling.

or something?